src/UsgsAstroLsSensorModel.cpp

```diff
@@ -317,8 +317,8 @@ csm::ImageCoord UsgsAstroLsSensorModel::groundToImage(
             "Divide by zero.",
             "UsgsAstroLsSensorModel::groundToImage");
       }
-      lineTemp += (dLine / det / DELTA_IMAGE);
-      sampleTemp += (dSamp / det / DELTA_IMAGE);
+      lineTemp += (dLine / det * DELTA_IMAGE);
+      sampleTemp += (dSamp / det * DELTA_IMAGE);

```

Comment:
This was a math error that was present in the original code. DELTA_IMAGE was just always 1, so it didn't matter.